environments/example-credentials/services/scoringmp/base/config/deployment.yaml

```diff
@@ -68,7 +68,7 @@ spec:
           #############################
           ### Reactive Messaging topics
           #############################
-          - name: MP_MESSAGING_INCOMING_REEFERTELEMETRY_TOPIC
+          - name: MP_MESSAGING_INCOMING_REEFER_TELEMETRY_TOPIC
```

Comment:
Is this meant to be part of this PR?

Reply:
Well, this is sth I though I had addressed in a previous PR but it seems I missed that one above and included in this PR so that we dont have just a PR with a one character change but logically it does not have anything to do with the SCRAM credentials work